core/arch/riscv64/riscv64.asm

```diff
@@ -368,8 +368,7 @@ GLOBAL_LABEL(dr_try_start:)
         DECLARE_FUNC(dr_setjmp)
```

Comment:
Per the commit messages section at https://dynamorio.org/page_code_reviews.html: we prefer to not keep the PR description empty. Please add a description of the changes made here.

Reply:
Done